python/ray/util/spark/databricks_hook.py

```diff
@@ -145,3 +145,7 @@ def auto_shutdown_watcher():
                 time.sleep(DATABRICKS_AUTO_SHUTDOWN_POLL_INTERVAL_SECONDS)
 
         threading.Thread(target=auto_shutdown_watcher, daemon=True).start()
+
+    def on_spark_job_created(self, job_group_id):
+        db_api_entry = get_db_entry_point()
+        db_api_entry.registerBackgroundSparkJobGroup("job_group_id")
```

Comment:
This is for registering spark job that will be killed when databricks notebook is detached.